scripts/data/PublishWidgetData.ts

```diff
@@ -0,0 +1,90 @@
+import * as fs from "fs";
+import { join, dirname } from "path";
+import { FileReadError, PatternNotFoundError } from "./errors";
+
+const result = ["packages/pluggableWidgets", "packages/customWidgets"]
```

Comment:
Maybe you'd like to rely on lerna's package cmd to find and then filter packages? Here's an example in the repo: https://github.com/mendix/widgets-resources/blob/master/scripts/validation/validate-versions-staged-files.js#L16

Reply:
Done. Thanks for the suggestion.

I made the whole thing async along the way. Not sure if that makes it better :/

Reply:
Nice one!